src/shared-endpoints.ts

```diff
@@ -0,0 +1,135 @@
+//type Constructor = new (...args: any[]) => {};
+import { GenericAPIResponse } from './util/requestUtils';
+import RequestWrapper from './util/requestWrapper';
+
+export default class SharedEndpoints {
+    protected requestWrapper: RequestWrapper; // XXX Is there a way to say that Base has to provide this?
+    
+    //------------Market Data Endpoints------------>
+    
+    getOrderBook(params: {
+        symbol: string;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/orderBook/L2', params);
+    }
+    
+    /**
+    * @deprecated use getTickers() instead
+    */
+    getLatestInformation(params?: {
+        symbol?: string;
+    }): GenericAPIResponse {
+        return this.getTickers(params);
+    }
```

Comment:
Since this is deprecated, I would move `getLatestInformation()` back to the inverse client (where some implementations may be using the deprecated function). 

The `getTickers` can stay here as a shared endpoint, so any new implementations over the linear client will start using that, and in a future release we can remove getLatestInformation completely.